gimie/sources/git.py

```diff
@@ -72,9 +78,20 @@ def extract(self):
         self.contributors = self._get_contributors()
         self.date_created = self._get_creation_date()
         self.date_modified = self._get_modification_date()
+        self.license = self._get_licenses()
 
     def list_files(self) -> List[LocalResource]:
-        raise NotImplementedError
+        file_list = []
+
+        if self.local_path is None:
+            return file_list
+
+        for p in Path(self.local_path).rglob("*"):
+            if (p.parts[0] == ".git") or not p.is_file():
```

Comment:
is this a pythonic best practice to call a path p? like index i? (after finding out what r.glob does, I understand, but maybe it's not super readble?)

Reply:
fair enough, fixed in [53ddd90](https://github.com/SDSC-ORD/gimie/pull/78/commits/53ddd904734afbb0bb12138dee38a77116a3963a)